### PR TITLE
RND-146: Updated Run Workflow modal documentation

### PR DESCRIPTION
**Pull request by Vorbert-Kruk**
Documentation update is related to the changes introduced within [cloudify-stage#2521](https://github.com/cloudify-cosmo/cloudify-stage/pull/2521)

### CODE REVIEW

File: content/working_with/console/widgets/deploymentsView.md [hunk 1]
```diff
@@ -283,6 +283,9 @@ The _Run Workflow_ bulk action will show a modal that allows selecting a
 workflow that will be executed on each deployment that matches the current
 filter.
 
+After selecting workflow, fields allowing to modify values of the workflow parameters will appear.   
+Fields will be initialized with default values of the workflow parameters.
```
Comment (qooban) [inline content/working_with/console/widgets/deploymentsView.md:287]: What do you think about rephrasing it a bit:
```suggestion
After selecting a workflow, user will be able to adjust workflow parameters. Only simple type parameters (<list of supported types goes here>) which are common for deployments matching the current filter are presented. Parameters with default values will have the initial value set.
```
?

File: content/working_with/console/widgets/deploymentsView.md [hunk 2]
```diff
@@ -283,6 +283,9 @@ The _Run Workflow_ bulk action will show a modal that allows selecting a
 workflow that will be executed on each deployment that matches the current
 filter.
```
Comment (qooban) [inline content/working_with/console/widgets/deploymentsView.md:285]: Optional: 

Maybe we should also say something about the workflows listing - that only workflows common for all deployments matching the current filter are allowed to use? 🤔

Comment (Vorbert-Kruk) [reply content/working_with/console/widgets/deploymentsView.md:285]: Sure, seems like a nice idea 👍

File: content/working_with/console/widgets/deploymentsView.md (hunk 1 above)
Comment (Vorbert-Kruk) [reply content/working_with/console/widgets/deploymentsView.md:287]: Sure, seems like a nice adjustment 🎉

Comment (Vorbert-Kruk) [reply content/working_with/console/widgets/deploymentsView.md]: Adjusted within 6963aaf51124c3bd03f773a8eb4e1a9bbb1c31a6

File: content/working_with/console/widgets/deploymentsView.md (hunk 2 above)
Comment (Vorbert-Kruk) [reply content/working_with/console/widgets/deploymentsView.md:284]: Adjusted within 6963aaf51124c3bd03f773a8eb4e1a9bbb1c31a6

**Review by qooban — APPROVED** [review]

**Pull request merged**